core/tests/protractor_utils/gallery.js

```diff
@@ -96,6 +96,23 @@ var getExplorationObjective = function(name) {
   });
 };
 
+var expectExplorationRatingToEqual = function(name, ratingValue) {
+  _getExplorationElements(name).then(function(elems) {
+    elems[0].element(by.css('.protractor-test-exp-summary-tile-rating')).
+      getText().then(function(value) {
+        expect(value).toBe(ratingValue);
+    });
+  });
+};
+
+var expectExplorationRatingToBeNotDisplayed = function(name) {
```

Comment:
I would rename this to expectExplorationRatingToBeHidden() for consistency with other places in the code.